packages/maskbook/src/resources/SliderIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import { SvgIcon, SvgIconProps } from '@material-ui/core'
+
+const SliderIconSVG = (
```

Comment:
Hi please add the new icon to the `packages/icons` to make it shared across the whole repo.

To depend on that package, you should:

1. Add "@dimensiondev/icons": "*" at `packages/maskbook`
2. Add project reference at `packages/maskbook/tsconfig.json`

Reply:
And please add icons via a small PR. Because icons change will cause netlify to build a preview and that's costy

Reply:
@guanbinrui #2427 has merged. please use icon from that package thanks

Reply:
Got it. I'm removing this icon. cause it's not using anymore.